sematic/api/endpoints/runs.py

```diff
@@ -266,11 +266,26 @@ def get_logs_endpoint(user: Optional[User], run_id: str) -> flask.Response:
     if "filter_string" in kwarg_overrides:
         filter_string: str = kwarg_overrides["filter_string"]  # type: ignore
         kwarg_overrides["filter_strings"] = [filter_string]
-    default_kwargs = dict(continuation_cursor=None, max_lines=100, filter_strings=None)
+    if "continuation_cursor" in kwarg_overrides:
+        # For backwards compatibility; parameter used to be called
+        # continuation_cursor.
+        kwarg_overrides["continuation_cursor_token"] = kwarg_overrides[
+            "continuation_cursor"
+        ]
+        del kwarg_overrides["continuation_cursor"]
+    default_kwargs = dict(
+        continuation_cursor_token=None,
+        reverse_cursor_token=None,
+        max_lines=100,
+        filter_strings=None,
+        reverse=False,
+    )
     kwarg_converters = dict(
-        continuation_cursor=lambda v: v if v is None else str(v),
+        continuation_cursor_token=lambda v: v if v is None else str(v),
+        reverse_cursor_token=lambda v: v if v is None else str(v),
         max_lines=int,
         filter_strings=lambda v: [] if v is None else list(v),
+        reverse=lambda v: v if isinstance(v, bool) else v.lower() == "true",
```

Comment:
```
$ python3 -c "print(bool('False'))"
True
```

All non-empty strings are truthy.

Reply:
`default_kwargs` has it as a boolean. If the http request specified a value, it would be a string. This mechanism ensures that regardless of where the value came from it will be a bool in the end.